HWMgmt/OCPBaselineRedfishService.json

```diff
@@ -0,0 +1,571 @@
+            ],
+            "PropertyRequirements": {
+                "MACAddress": {},
+                "SpeedMbps": {
+                    "ReadRequirement": "Supported"
+                },
+                "InterfaceEnabled": {},
+                "LinkStatus": {},
+                "Status": {
+                    "PropertyRequirements": {
+                        "Health": {},
+                        "State": {}
+                    }
+                },
+                "DHCPv4": {
+                    "ReadRequirement": "Recommended",
+                    "WriteRequirement": "Recommended",
+                    "PropertyRequirements": {
+                        "DHCPEnabled": {
+                            "ReadRequirement": "IfImplemented"
+                        }
+                    }
+                },
+                "DHCPv6": {
+                    "ReadRequirement": "Recommended",
+                    "WriteRequirement": "Recommended",
+                    "PropertyRequirements": {
+                        "OperatingMode": {
+                            "ReadRequirement": "IfImplemented"
+                        }
+                    }
+                },
+                "HostName": {
+                    "ReadRequirement": "Supported"
+                },
+                "FQDN": {
+                    "ReadRequirement": "Supported"
+                },
+                "NameServers": {},
+                "IPv4Addresses": {
+                    "Address": {},
+                    "AddressOrigin": {},
+                    "Gateway": {},
+                    "SubnetMask": {}
+                },
+                "IPv4StaticAddresses": {
+                    "ReadRequirement": "Recommended"
+                },
+                "IPv6AddressPolicyTable": {
+                    "ReadRequirement": "Recommended"
+                },
+                "IPv6StaticAddresses": {
+                    "ReadRequirement": "Recommended"
+                },
+                "IPv6StaticDefaultGateways": {
+                    "ReadRequirement": "Recommended"
+                },
+                "IPv6Addresses": {
+                    "ReadRequirement": "IfImplemented",
+                    "PropertyRequirements": {
+                        "Address": {},
+                        "PrefixLength": {},
+                        "AddressOrigin": {},
+                        "AddressState": {}
+                    }
+                },
+                "StaticNameServers": {
+                    "ReadRequirement": "Recommended"
+                }
+            }
+        },
+        "EventDestination": {
+            "PropertyRequirements": {
+                "Context": {},
+                "Destination": {},
+                "DeliveryRetryPolicy": {},
+                "Protocol": {},
+                "EventFormatType": {},
+                "SubscriptionType": {},
+                "ExcludeMessageIds": {
+                    "ReadRequirement": "Recommended"
```

Comment:
Should these "Recommended" ones be "Supported" or "IfImplemented"? EventDestinations are client-specified, so a system under test won't necessarily have these readily shown in the service if a user never specified a subscription with these options.

Reply:
For validation it should have the same results, correct?  Supported would require the service to implement (must show at least once), and these aren't HW-driven features so it's not an if-implemented case...